src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -56,13 +56,14 @@
         "symfony/notifier": "^6.4|^7.0",
         "symfony/process": "^6.4|^7.0",
         "symfony/rate-limiter": "^6.4|^7.0",
+        "symfony/runtime": "^6.4.13|^7.1.6",
         "symfony/scheduler": "^6.4.4|^7.0.4",
         "symfony/security-bundle": "^6.4|^7.0",
         "symfony/semaphore": "^6.4|^7.0",
         "symfony/serializer": "^7.1",
         "symfony/stopwatch": "^6.4|^7.0",
         "symfony/string": "^6.4|^7.0",
-        "symfony/translation": "^6.4|^7.0",
+        "symfony/translation": "^6.4.3|^7.1",
```

Comment:
this is required to have a version of the `PhpAstExtractor` class that works with `nikic/php-parser` 5 (we need the changes from #53453